server/src/main/java/org/opensearch/action/search/SearchRequestOperationsListener.java

```diff
@@ -31,11 +31,11 @@ protected SearchRequestOperationsListener(final boolean enabled) {
         this.enabled = enabled;
     }
 
-    protected abstract void onPhaseStart(SearchPhaseContext context);
+    protected abstract void onPhaseStart(SearchPhaseContext context, SearchRequestContext searchRequestContext);
```

Comment:
Why do you adding `searchRequestContext`? Could you please point out where it is used?